geometry_msgs/Point face3d
geometry_msgs/Vector3 face6d
linux_kinect/Bit face2d
bool speaking
bool looking
